packages/lage/src/config/getConfig.ts

```diff
@@ -76,6 +76,9 @@ export function getConfig(cwd: string): Config {
     continue: parsedArgs.continue || configResults?.config.continue,
     safeExit: parsedArgs.safeExit,
     includeDependencies: parsedArgs.includeDependencies,
+    includeDependents: parsedArgs.includeDependents,
+    withDevDependencies: parsedArgs.withDevDependencies,
+    withPeerDependencies: parsedArgs.withPeerDependenies,
     clear: parsedArgs.clear || false,
```

Comment:
Default value question. :) When lage runs without any flags, what do each of these default to? (My guess is false?)

Reply:
The default value for a boolean in Typescript would be false, right? This is how PackageGraphFilter is defined in createPackageGraph within workspace-tools:

export interface PackageGraphFilter {
  namePatterns: string[];
  includeDependencies?: boolean;
  includeDependents?: boolean;
  withDevDependencies?: boolean;
  withPeerDependencies?: boolean;
}

Reply:
For actually running the commands, e.g. `lage build --scope pkg-a`:

devdeps should default to true, 
while the peerdeps mean very little in this context - and should be false (peer deps are usually things that are provided by something external, but I suppose SOMETIMES you are lucky that your dependencies are telling you to dep on something that already exists in your monorepo)

Reply:
In this case, defaulting devDeps to true please:

```ts
withDevDependencies: parsedArgs.withDevDependencies !== false
```